newsletter-august-2021.rst

```diff
@@ -0,0 +1,101 @@
+.. post:: August 2, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features in August.
+
+Read the Docs newsletter - August 2021
+======================================
+
+Welcome to a new edition of our monthly newsletter, where we
+share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- We have a new team member! Ana_ just joined us as Frontend Developer
+  in the context of :doc:`the CZI grant we were awarded </czi-grant-announcement/>`
+  to work on our JavaScript documentation embedding client
+  and the Read the Docs redesign and UX.
+- The first version of `the tutorial we contributed to the official
+  Sphinx documentation <https://www.sphinx-doc.org/en/master/tutorial/>`_ is online,
+  as part of the new Sphinx 4.1 releases, and we are already working on expanding it.
+  We thank the Sphinx maintainers for the thorough reviews
+  and CZI for making it possible.
+- `We are now tracking changes in our
+  database <https://github.com/readthedocs/readthedocs.org/pull/8355/>`_, which is
+  the first step in our roadmap to offer audit tracking on our platform.
+
+New features
+------------
+
+- Our commercial site `now allows projects to go public rather than private
+  only <https://docs.readthedocs.io/en/stable/versions.html#privacy-levels>`_,
+  and `to control the privacy level of pull request
+  builds <https://docs.readthedocs.io/en/stable/pull-requests.html#privacy-levels>`_
+  as well. After this, we will drop privacy levels for projects on our community site,
+  since they used to be a source of confusion.
```

Comment:
we kind of already did this, we are just missing migrating some private projects to public, new projects can't change that. Don't remember if we have mentioned this change publically before.

Reply:
Thanks, will put this in past tense anyway